src/manifest.json

```diff
@@ -21,7 +21,7 @@
   "host_permissions": ["*://*/*"],
   "permissions": ["storage"],
   "content_security_policy": {
-    "extension_pages": "script-src 'self' 'wasm-unsafe-eval'; object-src 'self'; frame-src 'none'; frame-ancestors 'none';"
+    "extension_pages": "script-src 'self' 'wasm-unsafe-eval'; object-src 'none'; default-src 'none'; connect-src *; style-src 'unsafe-inline'; frame-src 'none'; frame-ancestors 'none'; img-src 'self' data: https:;  media-src 'self' data: https:; "
```

Comment:
could we have a brief explanation on this?

Reply:
this was added in response to the incorrect password issue that happened while testing and for some user when updating the extension for more context on the issue itself please check the Gotcha section form here [Gotchas](https://www.stackhawk.com/blog/react-content-security-policy-guide-what-it-is-and-how-to-enable-it/)

Reply:
nice one!